buildSrc/src/main/groovy/bio.terra.pipelines.java-common-conventions.gradle

```diff
@@ -27,14 +27,14 @@ java {
 repositories {
     maven {
         // Terra proxy for maven central
-        url 'https://us-central1-maven.pkg.dev/dsp-artifact-registry/maven-central/'
+        url = 'https://us-central1-maven.pkg.dev/dsp-artifact-registry/maven-central/'
```

Comment:
something must have changed very recently with the deprecation of groovy stuff.  I couldnt get my tests running locally (even though i had run them 30 min prior) without these changes.  Error i was getting

```
Space-assignment syntax in Groovy DSL has been deprecated.`bio.terra.pipelines.java-common-conventions
```